src/main.rs

```diff
@@ -1,3 +1,22 @@
+use select::document::Document;
+use select::predicate::Name;
+
 fn main() {
+  get_header();
+}
+
+async fn get_header() {
+//    let test_url = Url::parse("https://example.net")?;
+//    println!("test_url scheme: {}\n    host  : {}\n    query : {}\n",
+//            test_url.scheme(), test_url.host_str(), test_url.query());
+
+  let res = reqwest::get("https://www.rust-lang.org/en-US/")
+      .await?
```

Comment:
got an error for ?
`cannot use the `?` operator in an async function that returns `()``

Reply:
solved